src/main/java/cpsc/dlsproject/server/Server.java

```diff
@@ -64,6 +79,41 @@ public static Builder newBuilder() {
     return new Builder();
   }
 
+  public void setupLoggingEndpoint() {
+    HttpHandler handler =
+        (httpExchange) -> {
+          this.increaseEndpointHitFrequency(this.getLoggingEndpointUrl());
```

Comment:
Why are we doing this for the log endpoint again if we're setting it to zero at the end anyway?

Reply:
We are increasing the frequency in the handler method, that is not called when setting up the logging endpoint. The handler method is only called when a client calls/makes a request to the endpoint method